CRITICAL - 2023-05-03 00:02:03 --> Trying to access array offset on value of type null
in APPPATH\Controllers\Auth.php on line 66.
 1 APPPATH\Controllers\Auth.php(66): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\gfp\\app\\Controllers\\Auth.php', 66)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->AutenticarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-03 00:04:31 --> Trying to access array offset on value of type null
in APPPATH\Controllers\Auth.php on line 66.
 1 APPPATH\Controllers\Auth.php(66): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\gfp\\app\\Controllers\\Auth.php', 66)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->AutenticarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-03 00:04:34 --> Trying to access array offset on value of type null
in APPPATH\Controllers\Auth.php on line 66.
 1 APPPATH\Controllers\Auth.php(66): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\gfp\\app\\Controllers\\Auth.php', 66)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->AutenticarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-03 00:07:43 --> Undefined array key "a3333@gmail.com"
in APPPATH\Controllers\Auth.php on line 64.
 1 APPPATH\Controllers\Auth.php(64): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "a3333@gmail.com"', 'C:\\xampp\\htdocs\\gfp\\app\\Controllers\\Auth.php', 64)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->AutenticarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-03 00:09:28 --> Call to undefined method App\Models\EmailsModel::Auth_usuario
in SYSTEMPATH\Model.php on line 842.
 1 APPPATH\Controllers\Auth.php(59): CodeIgniter\Model->__call('Auth_usuario', [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->AutenticarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-03 00:20:35 --> mysqli_sql_exception: Duplicate entry 'Mila' for key 'usuario' in C:\xampp\htdocs\gfp\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\gfp\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\gfp\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\gfp\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\gfp\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\gfp\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\gfp\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\gfp\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\gfp\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\gfp\app\Controllers\Auth.php(100): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\gfp\system\CodeIgniter.php(934): App\Controllers\Auth->guardar()
#10 C:\xampp\htdocs\gfp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#11 C:\xampp\htdocs\gfp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\gfp\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-05-03 00:20:35 --> Duplicate entry 'Mila' for key 'usuario'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `usuarios` (`usuario`, `nombre`, `apellido`, `tipo_documento`, `num_documento`, `pass`, `fecha_crea`) VALUES (\'Mila\', \'Miladys Esther\', \'Olivares Navarro\', \'9\', \'22549431\', \'$2y$10$R55sR4T7u.6bn/tPHQ9youU7hWU54geYcG6XiuFitLa6uBxwyX7X.\', \'2023-05-03 00:20:35\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuario`, `nombre`, `apellido`, `tipo_documento`, `num_documento`, `pass`, `fecha_crea`) VALUES (\'Mila\', \'Miladys Esther\', \'Olivares Navarro\', \'9\', \'22549431\', \'$2y$10$R55sR4T7u.6bn/tPHQ9youU7hWU54geYcG6XiuFitLa6uBxwyX7X.\', \'2023-05-03 00:20:35\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuario`, `nombre`, `apellido`, `tipo_documento`, `num_documento`, `pass`, `fecha_crea`) VALUES (\'Mila\', \'Miladys Esther\', \'Olivares Navarro\', \'9\', \'22549431\', \'$2y$10$R55sR4T7u.6bn/tPHQ9youU7hWU54geYcG6XiuFitLa6uBxwyX7X.\', \'2023-05-03 00:20:35\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuario`, `nombre`, `apellido`, `tipo_documento`, `num_documento`, `pass`, `fecha_crea`) VALUES (:usuario:, :nombre:, :apellido:, :tipo_documento:, :num_documento:, :pass:, :fecha_crea:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Auth.php(100): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->guardar()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-03 00:25:12 --> Array to string conversion
in APPPATH\Controllers\Auth.php on line 58.
 1 APPPATH\Controllers\Auth.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\xampp\\htdocs\\gfp\\app\\Controllers\\Auth.php', 58)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->AutenticarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-03 00:26:22 --> Array to string conversion
in APPPATH\Controllers\Auth.php on line 58.
 1 APPPATH\Controllers\Auth.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\xampp\\htdocs\\gfp\\app\\Controllers\\Auth.php', 58)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->AutenticarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-03 00:34:22 --> Undefined variable $id
in APPPATH\Controllers\Auth.php on line 63.
 1 APPPATH\Controllers\Auth.php(63): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $id', 'C:\\xampp\\htdocs\\gfp\\app\\Controllers\\Auth.php', 63)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->AutenticarUsuario()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-03 00:35:05 --> syntax error, unexpected token ",", expecting "]"
in APPPATH\Controllers\Auth.php on line 65.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\gfp\\app\\Controllers\\Auth.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Auth')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Auth')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\Auth', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
